core/lib/spree/core/engine.rb

```diff
@@ -55,6 +56,12 @@ class Engine < ::Rails::Engine
         app.config.spree.line_item_comparison_hooks = Set.new
       end
 
+      initializer 'spree.register.address_validators', before: :load_config_initializers do |app|
```

Comment:
@brozek95 let's use the same method to set default values as we have with payment methods, calculators, etc